common/client/versionChecker.go

```diff
@@ -40,11 +40,11 @@ const (
 	CLI = "cli"
 
 	// SupportedGoSDKVersion indicates the highest go sdk version server will accept requests from
-	SupportedGoSDKVersion = "1.6.0"
+	SupportedGoSDKVersion = "1.7.0"
 	// SupportedJavaSDKVersion indicates the highest java sdk version server will accept requests from
-	SupportedJavaSDKVersion = "1.6.0"
+	SupportedJavaSDKVersion = "1.7.0"
```

Comment:
This should actually be 1.4.0 as specified in the java client. The previously version was not correct. I have already updated the version and landed to master, you can simply merge my changes.